src/main/java/edu/ksu/canvas/model/Course.java

```diff
@@ -118,6 +119,15 @@ public void setName(String name) {
         this.name = name;
     }
 
+    @CanvasField(postKey = "created_at")
```

Comment:
It shouldn't have a postKey as we shouldn't be sending this field back to Canvas and so it's misleading.

Reply:
Sure.